apps/omg_watcher/lib/omg_watcher/exit_processor/core.ex

```diff
@@ -460,17 +459,17 @@ defmodule OMG.Watcher.ExitProcessor.Core do
   @spec check_validity(ExitProcessor.Request.t(), t()) :: check_validity_result_t()
   def check_validity(
         %ExitProcessor.Request{
-          eth_height_now: eth_height_now,
+          eth_timestamp_now: eth_timestamp_now,
           utxos_to_check: utxos_to_check,
           utxo_exists_result: utxo_exists_result,
           blocks_result: blocks
         },
         %__MODULE__{} = state
       )
-      when not is_nil(eth_height_now) do
+      when not is_nil(eth_timestamp_now) do
```

Comment:
this guard is irrelevant and should be removed!

Reply:
removed `96d9e31`